src/main/java/com/dataloom/edm/EdmApi.java

```diff
@@ -61,12 +63,24 @@
     EntityDataModel getEntityDataModel();
 
     /**
-     * Creates a schema.
+     * Creates an empty schema with default ACL_ID.
```

Comment:
That has deprecated for a long time. Can @geekbeast confirm? I am not considering removal of Acl Ids to be of high priority, and am planning to remove Acl Ids later once other tasks are done.